west.yml

```diff
@@ -140,6 +142,12 @@ manifest:
       path: modules/alexa-embedded
       revision: face92d8c62184832793f518bb1f19379538c5c1
       remote: alexa
+    - name: sdk-homekit
+      path: homekit
```

Comment:
Just checking that we really want this in the NCS workspace topdir. That seems reasonable to me since the user has to explicitly make the project active, so they must be interested in it and using it frequently.

Reply:
The sample and application files are in homekit repository. So I think the user will be interested in using it frequently.

Reply:
Thanks for the update. I agree. That means this line can be deleted now that the `name: homekit` line has been added. The `path` value defaults to the `name` value if not explicitly provided.

Reply:
Done